tests/testsuite/git_shallow.rs

```diff
@@ -9,11 +9,29 @@ enum Backend {
     Gitoxide,
 }
 
+impl Backend {
+    fn to_arg(&self) -> &'static str {
```

Comment:
nit: this method name seems too general? We may have features like checkout or status some day.